dmutils/email.py

```diff
@@ -91,16 +87,16 @@ def send_email(to_email_addresses, email_body, subject, from_email, from_name, r
         return_address = current_app.config.get('DM_EMAIL_RETURN_ADDRESS')
 
         result = email_client.send_email(
-            Source=u"{} <{}>".format(from_name, from_email),
+            Source="{} <{}>".format(from_name, from_email),
             Destination=destination_addresses,
             Message={
                 'Subject': {
-                    'Data': subject,
+                    'Data': subject.decode(),
                     'Charset': 'UTF-8'
                 },
                 'Body': {
                     'Html': {
-                        'Data': email_body,
+                        'Data': email_body.decode(),
```

Comment:
No need to decode()

Reply:
@tobiasbrown should we remove `email_body = to_bytes(email_body)` that way we can remove `'Data': email_body.decode()`

Reply:
Let's leave this as is. Emails have been received during testing.